inventory-aws

```diff
@@ -20,11 +20,15 @@
 10.128.1.207    internal_ip=10.128.1.207  name=tsuru-docker
 
 [hipache]
-10.128.1.127  internal_ip=10.128.1.127  external_ip=hipache.tsuru.paas.alphagov.co.uk name=tsuru-router
-10.128.3.65     internal_ip=10.128.3.65     external_ip=hipache.tsuru.paas.alphagov.co.uk name=tsuru-router
+10.128.1.127    internal_ip=hipache-int.tsuru.paas.alphagov.co.uk  external_ip=hipache.tsuru.paas.alphagov.co.uk name=tsuru-router
+10.128.3.65     internal_ip=hipache-int.tsuru.paas.alphagov.co.uk  external_ip=hipache.tsuru.paas.alphagov.co.uk name=tsuru-router
```

Comment:
Tiny nitpick, I think the commit message for this is slightly off:

> Using the load balancers host entry instead of its IP address. This is
> best practice for Amazon EC2 while not breaking the current vagrant
> setup.

These were the IPs of individual instances rather than an ELB and `hipache_host_internal_lb` (which wasn't actually used) was only pointing to the first of them. By changing it to the CNAME which points at the ELB we're now using both of them.